source/getting-started/index.md

```diff
@@ -1,52 +1,55 @@
 Getting started with Ember.js is easy. Ember.js projects are created and managed
-through our [command line build tool ember-cli](http://www.ember-cli.com/). The command line tool brings:
-
-* Modern application asset management (including combining, minifying, and versioning).
-* Built-in generators to help you create components, routes, and more.
-* A conventional project layout so approaching other developers' Ember.js applications is easy.
-* Official [JavaScript modules](http://jsmodules.io/) to keep your project organized.
-* A complete testing framework.
-* Access to a growing ecosystem of [Ember Addons](http://www.emberaddons.com/).
-
+through our command line build tool [Ember CLI](http://www.ember-cli.com/).
+This tool provides:
+
+* Modern application asset management (including concatenation, minifying, and versioning)
+* Generators to help you create components, routes, and more
+* A conventional project layout, so approaching existing Ember.js applications is easy
+* Support for ES2015/ES6 JavaScript via the [Babel](http://babeljs.io/docs/learn-es2015/)
+  project. This includes support for JavaScript modules, which are used
```

Comment:
http://eviltrout.com/2014/05/03/getting-started-with-es6.html is pretty good, but we decided not to link to specific blog posts/talks/slides, because the review overhead is too much. http://jsmodules.io/ is not great, but may be the best thing for here...